joblib/func_inspect.py

```diff
@@ -44,29 +37,14 @@ def get_func_code(func):
     more robust.
     """
     source_file = None
+    # Ignore wrappers.
+    while hasattr(func, "__wrapped__"):
```

Comment:
Can you explain why this is needed? Previous attempts like https://github.com/joblib/joblib/pull/1550/files seem to not do it for some reason ...

Reply:
OK thanks for the details!

Just to double-check, the `__wrapped__` attribute exists where `functools.wrap` right?

Reply:
> Just to double-check, the `__wrapped__` attribute exists where `functools.wrap` right?

Exactly!